app/src/main/res/values-nl/strings.xml

```diff
@@ -97,6 +97,13 @@
     <string name="contact_set_name">Geef contact een naam</string>
     <string name="contact_exists_message">Contact bestaat al, geef aub een unieke naam</string>
 
+    <!-- NFC -->
+    <string name="descr_gif">Dit zou een gif moeten zijn die nfc connection uitbeeldt.</string>
```

Comment:
@clenrock told me this should describe what the view was depicting so I think it does a good job on that 🙈